packages/office-ui-fabric-react/src/components/Button/CommandButton/CommandButton.tsx

```diff
@@ -27,9 +28,10 @@ export class CommandButton extends BaseComponent<IButtonProps, {}> {
   public render() {
     return (
       <BaseButton
-        classNames={ CLASS_NAMES }
+        { ...this.props }
+        classNames={ assign({}, CLASS_NAMES, this.props.classNames) }
```

Comment:
The assign is to allow prop classnames to override the button specific class names. i.e. I want a new .isEnabled:hover style, I can override their isEnabled with my own. And since it's css modules, even if I use the same isEnabled class name, the hash will be different.